BrowserKit/Sources/ComponentLibrary/Buttons/CloseButton.swift

```diff
@@ -17,6 +22,8 @@ public class CloseButton: UIButton {
         super.init(frame: frame)
 
         setImage(UIImage(named: UX.crossCircleImage), for: .normal)
+        adjustsImageSizeForAccessibilityContentSizeCategory = true
+        imageView?.adjustsImageSizeForAccessibilityContentSizeCategory = true
```

Comment:
Yes just the first one, the second can be removed. It's needed since when scaling to bigger fonts we are updating only the frame of the button and the image remains the same of smaller font sizes